nima/inference/inference_model.py

```diff
@@ -34,7 +37,11 @@ def predict(self, image):
         image = self.transform(image)
         image = image.unsqueeze_(0)
         image = torch.autograd.Variable(image, volatile=True)
-        prob = self.model(image).data.numpy()[0]
+        if self.use_gpu:
+            image = image.cuda()
+            prob = self.model(image).data.cpu().numpy()[0]
```

Comment:
@abliznyuk 
1) repeated code 
2) also will not work. because .numpy() method for gpu tensor will cause error 
"can't convert CUDA tensor to numpy"
you need to move the tensor to host memory, with .cpu() method